src/state/pools/fetchPools.ts

```diff
@@ -51,26 +52,25 @@ export const fetchPoolsBlockLimits = async () => {
   })
 }
 
-export const fetchPoolsTotalStaking = async () => {
-  const nonBnbPools = poolsConfig.filter((p) => p.stakingToken.symbol !== 'BNB')
-  const bnbPool = poolsConfig.filter((p) => p.stakingToken.symbol === 'BNB')
-
-  const callsNonBnbPools = nonBnbPools.map((poolConfig) => {
-    return {
-      address: poolConfig.stakingToken.address,
-      name: 'balanceOf',
-      params: [getAddress(poolConfig.contractAddress)],
-    }
-  })
+const nonBnbPools = poolsConfig.filter((p) => p.stakingToken.symbol !== 'BNB')
```

Comment:
we can use lodash partition, something like this

`
 const [bnbPool, nonBnbPools] = partition(pools, (p) => p.stakingToken.symbol === 'BNB')
`